mkosi/config.py

```diff
@@ -1555,69 +1617,66 @@ def __init__(self) -> None:
         self.settings_lookup_by_dest = {s.dest: s for s in self.SETTINGS}
         self.match_lookup = {m.name: m for m in self.MATCHES}
 
-    def parse_config(self, path: Path, namespace: argparse.Namespace, defaults: argparse.Namespace) -> bool:
-        extras = path.is_dir()
+    def match_config(self, path: Path, namespace: argparse.Namespace, defaults: argparse.Namespace) -> bool:
+        triggered = None
 
-        if path.is_dir():
-            path = path / "mkosi.conf"
+        if not path.exists():
+            return True
```

Comment:
This part of the logic could use a comment, because with the `not` at the place of use in `parse_config` this is a whole lot of inversions, that I find a bit confusing